README.md

```diff
@@ -30,32 +30,30 @@ The marketplace operator manages two CRDs: [OperatorSource](./deploy/operatorsou
 
 The operator will then create a CatalogSourceConfig CR which will, for the time being, trigger the marketplace operator to create a ConfigMap CR and CatalogSource CR. The package-server, managed by [OLM](https://github.com/operator-framework/operator-lifecycle-manager), will then respond to the creation of these CRs and allow the external operators to be visible in the [marketplace UI](https://github.com/openshift/console/tree/master/frontend/public/components/marketplace).
 
-### Deploying the Marketplace Operator using Operator Artifacts
+### Deploying the Marketplace Operator with OKD
+It is important to note that the order in which you apply the Operator Artifacts matters, do not execute the `oc apply` commands featured in this section out of order.
 
-#### In an OKD Cluster
+#### Deploying the Marketplace Operator using Operator Artifacts
```

Comment:
Given the changes @SamiSousa is introducing it will better to remove `using Operator Artifacts` here,

Reply:
Completed the requested changes.